agent/agentbootstrap/bootstrap.go

```diff
@@ -108,11 +111,23 @@ func InitializeState(
 	info.Tag = nil
 	info.Password = c.OldPassword()
 
+	// Add the controller model cloud and credential to the database.
+	cloudCred, cloudCredTag, err := getCloudCredential(adminUser, args)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	opts := append(setupDBOpts,
+		ccbootstrap.InsertInitialControllerConfig(args.ControllerConfig),
+		cloudbootstrap.InsertInitialControllerCloud(args.ControllerCloud),
+	)
+	if cloudCredTag.Id() != "" {
```

Comment:
As per comment above I would prefer `!errors.Is(err, errors.NotFound)`

Reply:
My yak shaving razor is very blunt by now